### DISCUSSION TITLE
rainbow indentation for file tree explorer

### DISCUSSION THREAD

**Discussion by fMagVen** [microsoft/vscode-discussions#465] [Extension Development QnA] [answered]
Asked this on stackoverflow but it got rejected, sadly

Looking for an extension that can add rainbow indentation like this https://marketplace.visualstudio.com/items?itemName=oderwat.indent-rainbow

But for the file tree explorer, like this https://stackoverflow.com/questions/41898245/

**Comment by gjsjohnmurray** [comment, marked as answer]
I don't think this is possible with the current API.
